ui.apps/src/main/content/jcr_root/apps/saas-aem-module/clientlibs/wknd-sample/css/styles.less

```diff
@@ -52,61 +67,61 @@
   transition: 0.08s ease-in;
 }
 @media (prefers-color-scheme: dark) {
-  .search .saas-container_results_item {
+  .search .cmp-saas__results-item {
     background-color: #fff;
   }
 }
 @media only screen and (max-width: 768px) {
-  .search .saas-container_results_item {
+  .search .cmp-saas__results-item {
     margin: 15px 0 0;
   }
 }
 @media only screen and (min-width: 769px) {
-  .search .saas-container_results_item {
+  .search .cmp-saas__results-item {
     margin: 0 0 15px 15px;
   }
 }
 @media only screen and (min-width: 992px) {
-  .search .saas-container_results_item:hover {
+  .search .cmp-saas__results-item:hover {
     -webkit-box-shadow: 0 0 0 1px #ffea00;
     box-shadow: 0 0 0 1px #ffea00;
   }
 }
-.search .saas-container_results_item_description {
+.search .cmp-saas__results-item-description {
   color: dimgray;
   font-size: 14px;
   line-height: inherit;
   margin-top: 30px;
   text-align: start;
 }
-.search .saas-container_results_item_title {
+.search .cmp-saas__results-item-title {
   color: #202020;
   font-size: 20px;
   margin-bottom: 10px;
   margin-top: 10px;
 }
-.search .saas-container_results_item_title:after {
+.search .cmp-saas__results-item-title:after {
   border-bottom: 2px solid #ffea00;
   content: '';
   display: block;
   padding-top: 8px;
   width: 84px;
 }
-.search .saas-container_results_item_url {
+.search .cmp-saas__results-item-url {
   color: #202020;
   display: block;
   font-size: 14px;
   height: 100%;
   padding: 15px;
   word-break: break-word;
 }
-.search .saas-container_results_item_link_text {
+.search .cmp-saas__results-item-link-text {
   color: #0d66d0;
   display: none;
   font-size: 12px;
   word-break: break-all;
 }
-.search .saas-container_form input[placeholder] {
+.search .cmp-saas__form input[placeholder] {
```

Comment:
as we are on it - should we style html elements or rather add a class here?

Reply:
You are right, with class would be better. I will do the change. Thanks for that!